proxy-server/options/options.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="options.css"/>
+  </head>
+
+  <body>
+
+    <p>
+      <label>Proxy server DNS name or IP address</label>
+      <input id="proxyServerAddressInput"/><br>
+      <label>Proxy server port</label>
+      <input id="proxyServerPortInput"/><br>
+      <label>Proxy server username (optional)</label>
+      <input id="proxyServerUsernameInput"/><br>
+      <label>Proxy server password (optional)</label>
+      <input id="proxyServerPasswordInput"/><br>
+      <label>Proxy server type</label><br>
+      <input name="proxyServerTypeInput" type="radio" value="HTTP" checked>HTTP
+      <input name="proxyServerTypeInput" type="radio" value="HTTPS">HTTPS/SSL
+      <input name="proxyServerTypeInput" type="radio" value="SOCKS">SOCKS5
+    </p>
```

Comment:
This looks pretty rough, because the boxes are all aligned differently. It would look better if the boxes came before the labels: http://imgur.com/a/lxwa3